### PR TITLE
Add createAt timestamp for native tokens

### PR DESCRIPTION
**Pull request by hanyunx**
As mentioned in #21.

### CODE REVIEW

File: contracts/NonReservedNativeTokenManager.sol [hunk 1]
```diff
@@ -27,6 +27,7 @@ contract NonReservedNativeTokenManager {
     }
 
     struct NativeToken {
+        bool isTaken;
```
Comment (ninjaahhh) [inline contracts/NonReservedNativeTokenManager.sol]: `exists`?

Comment (ninjaahhh) [reply contracts/NonReservedNativeTokenManager.sol]: also, do we need a getter to get a certain native token's information?

Comment (hanyunx) [reply contracts/NonReservedNativeTokenManager.sol]: It would be good to have a `exists` getter. Are all elements of `NativeToken` need to be accessible?

Comment (ninjaahhh) [reply contracts/NonReservedNativeTokenManager.sol]: I think yes. all users should know the owner and total supply, right?

Comment (hanyunx) [reply contracts/NonReservedNativeTokenManager.sol]: Make sense!

File: contracts/NonReservedNativeTokenManager.sol [hunk 2]
```diff
@@ -27,6 +27,7 @@ contract NonReservedNativeTokenManager {
     }
 
     struct NativeToken {
+        uint128 createAt;
```
Comment (ninjaahhh) [inline contracts/NonReservedNativeTokenManager.sol]: now the concern becomes the storage: now `NativeToken` takes up 3 words. how about `uint64`?

**Pull request merged**